docs/json-schema/common.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Dropped.

The docs site can only do relative links to things within the docs site. If we want to avoid hard-coded URLs in the source tree we need to make sure that there is a relative target we point at. For now I've simply removed the links entirely.